docs/packages.rst

```diff
@@ -117,42 +125,59 @@ community.
 General Best Practices for Open Source
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-1. Use an open-source license. See `Jake VanderPlas' article on
-   licensing scientific
-   code <http://www.astrobetter.com/blog/2014/03/10/the-whys-and-hows-of-licensing-scientific-code/>`__
-   or `these more general guidelines <https://choosealicense.com/>`__
-2. Use version control for source code (for example, on
-   `github <http://github.org>`__)
-3. Provide thorough test coverage and continuous integration of testing
-4. Maintain comprehensive documentation
-5. Establish a `code of
-   conduct <https://opensource.guide/code-of-conduct/>`__ for
-   contributors
-
-The `open-source guide <https://opensource.guide/>`__ provides some
-great advice on building and maintaining open-source projects.
+How to develop and maintain an open-source project is a large topic that extends beyond pangeo.
+The `pyOpenSci Package Development Guide`_ provides a comprehensive guide.
+
+Projects can submit their package for peer review from pyOpenSci.
```

Comment:
Our peer review documentation however is much more stable. BUT i'm making some changes to the url paths. But htat will be implemented by mid next week. And all urls will have redirecits : 
https://www.pyopensci.org/peer-review-guide/index.html in case they want information on how to submit a package for review.

Reply:
i will have the pangeo standards language in this update next week as well